PluginCore/PluginCore/Helpers/PathHelper.cs

```diff
@@ -19,7 +20,7 @@ public static String BaseDir
         {
             get
             {
-                if (PluginBase.MainForm.StandaloneMode) return AppDir;
+                if (PluginBase.MainForm.StandaloneMode) return Path.GetDirectoryName(GetAssemblyPath(Assembly.GetExecutingAssembly()));
```

Comment:
If I fully understand all of what you say, I don't agree. If some part of code tries to load some file that should be relative to the base directory using the application directory, I would say it's wrong. As I say, I see AppDir and BaseDir as two different things, and there is no reason for them to be always the same thing in standalone mode.

I only changed the base dir not because it's related to ouput path, but because the snippet and template directories use the BaseDir as the root folder.

Anyway, take into account that in normal cases PluginCore and FlashDevelop.exe will always be in the same directory, so there won't be any noticeable change for any user.